src/query.rs

```diff
@@ -252,28 +281,32 @@ mod tests {
                         )
                     };
 
-                assert!(
-                    run_query_on(&indexed_crate_new, &indexed_crate_new)
-                        .1
-                        .is_empty(),
-                    "{}",
-                    crate_pair
-                );
-                assert!(
-                    run_query_on(&indexed_crate_old, &indexed_crate_old)
-                        .1
-                        .is_empty(),
-                    "{}",
-                    crate_pair
-                );
+                let assert_no_false_positives_in_nonchanged_crate =
+                    |crate_: &VersionedIndexedCrate,
+                     crate_version: &str| {
+                         let output_pair = run_query_on(crate_, crate_);
+                         if !output_pair.1.is_empty() {
+                             let output_difference = pretty_format_output_difference(
+                                 query_name,
+                                 "Expected output (empty output)".to_string(),
+                                 BTreeMap::new(),
+                                 format!("Actual output ({}/{})", crate_pair, crate_version),
```

Comment:
nit: I think the next `clippy` version is going to get lints that encourage inline use of the formatting variables whenever possible, like so:
```suggestion
                                 format!("Actual output ({crate_pair}/{crate_version})"),
```
I already had to fix a bunch of them earlier: https://github.com/obi1kenobi/cargo-semver-checks/pull/219/files#diff-f0c63cae3ed0328ebbe14ac22d86b43bc8a40146944b03c8326681e4f3802443L217-R221

Reply:
Fixed in c2222fffe7f3daa27fb468c6f542b1ce1665fac1